js/components/image.js

```diff
@@ -67,6 +68,10 @@ Fliplet.FormBuilder.field('image', {
       $(this.$refs.imageInput).parents('.form-group').removeClass('has-error');
 
       if (!this.required) {
+        if (this.value.length && !this.isImageValid) {
+          $(this.$refs.imageInput).parents('.form-group').addClass('has-error');
```

Comment:
the `$(this.$refs.imageInput).parents('.form-group').addClass('has-error');` line is duplicated, is there a way we can avoid this? e.g. using the `isImageValid` boolean to bind the class via the vue template?

Reply:
we will look into it.